fonts/OTF/TeX/makeFF

```diff
@@ -1517,11 +1517,12 @@ $map{cmtt10} = {
     0x15 => [0x306,-525,0], # \breve (combining)
     0x16 => [0x304,-525,0], # \bar (combining)
     0x17 => [0x30A,-525,0], # ring above (combining)
+    0x20 => 0x2423,         # graphic representation of space
```

Comment:
Don't we want the actual space character it's self?  `\texttt{a b}` in LaTeX puts a space between `a` and `b` instead of rendering a `␣`.